inst/create-test-analyses.R

```diff
@@ -1,6 +1,10 @@
 # NOTE: IF YOU RE-RUN THIS SCRIPT YOU WILL NEED TO UPDATE SNAPSHOTS
+# This script generates test analysis objects for the sensitivity functions.
 
 if (FALSE) {
+  library(jmbr)
```

Comment:
Can we use withr to just load for the duration of the test so as not to alter the search path?